scripts/ci_checkout_frontend.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+if [[ ! -z "${CIRCLE_BRANCH}" ]]; then
+  CI_BRANCH="${CIRCLE_BRANCH}"
+fi
+
+if [[ ! -z "${GITHUB_REF}" ]]; then
+  CI_BRANCH="${GITHUB_REF/refs\/heads\//}"
+fi
+
+CI_BRANCH=${CI_BRANCH:=master}
+
+TARBALL_URL="https://codeload.github.com/opencollective/opencollective-frontend/tar.gz/"
+echo "> Check ${TARBALL_URL}${CI_BRANCH}"
+if curl -s --head --request GET "${TARBALL_URL}${CI_BRANCH}" | grep "200" > /dev/null
```

Comment:
Nice that we keep matching branches :+1:

Reply:
I tried to implement that with a cleaner syntax using GitHub actions checkout. https://github.com/actions/checkout 

Was almost working but was ultimately failing because the first attempt was sending a CI failure. Will share a PR and open an issue on GitHub side to ask for that feature to be somehow supported.

Reply:
Related links:
- https://github.com/opencollective/opencollective-api/pull/3031
- https://github.com/actions/checkout/issues/105